jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/NavbarBrand.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.jdi.light.ui.bootstrap.elements.common;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.interfaces.base.IListBase;
+import com.epam.jdi.light.ui.bootstrap.interfaces.IsNavbarBrand;
+
+import static com.epam.jdi.light.logger.LogLevels.DEBUG;
+
+
+public class NavbarBrand extends UIBaseElement<UIAssert> implements IsNavbarBrand, IListBase {
```

Comment:
NavbarBrand is any custom element. You shouldn't create such element in bootstrap package, because it doesn't have any specific and unique properties or methods.

Reply:
Do not agree (at least for now). I think that any custom element has to have a possibility to  be collected from page using JList. Since common elements are not implemented proper interface for it I create navbar that can be taken from page to JList directly.

Reply:
> I think that any custom element has to have a possibility to be collected from page using JList.
But now there is not such possibility. To fix it, we have to change all custom elements, or change JList.

You can use UIElement  in JList, or you can use WebList:

```
public class NavbarSection extends Section {

    //FindBy(css = ".navbar-brand")
    @UI(".navbar-brand")
    public JList<UIElement> allNavbarBrands;

    //FindBy(css = "#brand-as-image,#brand-as-image-and-link")
    @UI("#brand-as-image,#brand-as-image-and-link")
    public WebList navbarBrandWithImage;

}
```